### PR TITLE
Removed SABER.

### PR DESCRIPTION
**Pull request by xvzcf**

### CODE REVIEW

File: scripts/copy_from_upstream/patches/pqclean-kyber-arm-macos-gcc-fix.patch [hunk 1]
```diff
@@ -0,0 +1,318 @@
...
+ .type PQCLEAN_KYBER1024_AARCH64_asm_intt_SIMD_bot, %function
+ #endif
+ PQCLEAN_KYBER1024_AARCH64_asm_intt_SIMD_bot:
+@@ -90,7 +90,7 @@ _PQCLEAN_KYBER1024_AARCH64_asm_intt_SIMD_bot:
+ .align 2
+ .global PQCLEAN_KYBER1024_AARCH64_asm_intt_SIMD_top
+ .global _PQCLEAN_KYBER1024_AARCH64_asm_intt_SIMD_top
+-#ifndef __clang__
++#if !defined(__clang__) && !defined(old_gas_syntax)
+ .type PQCLEAN_KYBER1024_AARCH64_asm_intt_SIMD_top, %function
```
Comment (baentsch) [inline scripts/copy_from_upstream/patches/pqclean-kyber-arm-macos-gcc-fix.patch:73]: This doesn't look like a SABER-related change. Where does this come from?

**Review by baentsch — CHANGES_REQUESTED** [review]
Please investigate/explain how the Kyber-compiler define changes sneaked into this PR.

File: scripts/copy_from_upstream/patches/pqclean-kyber-arm-macos-gcc-fix.patch (hunk 1 above)
Comment (xvzcf) [reply scripts/copy_from_upstream/patches/pqclean-kyber-arm-macos-gcc-fix.patch:73]: The patch was previously called `pqclean-saber-kyber-arm-macos-gcc-fix.patch`. I removed all the saber related changes, created a new patch called `pqclean-kyber-arm-macos-gcc-fix.patch`, and removed the old patch.

Comment (baentsch) [reply scripts/copy_from_upstream/patches/pqclean-kyber-arm-macos-gcc-fix.patch:73]: Ahh -- I was confused by the deltas -- but it's clearly a new file. Thanks for the explanation

**Comment by baentsch** [comment]
> Please investigate/explain how the Kyber-compiler define changes sneaked into this PR.

Rationale understood. Unrelated to the PR, what makes me wonder now is why this "kyber_pqclean" patch is not required when importing kyber straight from pqcrystals?

**Review by baentsch — APPROVED** [review]

**Comment by xvzcf** [comment]
> > Please investigate/explain how the Kyber-compiler define changes sneaked into this PR.
> 
> Rationale understood. Unrelated to the PR, what makes me wonder now is why this "kyber_pqclean" patch is not required when importing kyber straight from pqcrystals?

I believe this is for the ARM code, which we're still taking from PQClean.

**Pull request merged**